services/reader.go

```diff
@@ -88,9 +123,18 @@ func ReadColumns(file multipart.File, sheetsToExtract []string) (Output, error)
 				return nil, err
 			}
 
+			if currentColIndex < defaultColIndex {
+				currentColIndex++
+				continue
+			}
+
 			rowValue := make([]interface{}, 0)
 
-			for _, cellValue := range col {
+			for index, cellValue := range col {
+				if defaultRowIndex > 1 && index < (defaultRowIndex-1) {
```

Comment:
why `defaultRowIndex > 1`? If the initial row index is modified then it will automatically greater than 1, so you will never get its value